website/docs/r/frontdoor_custom_https_configuration.html.markdown

```diff
@@ -75,12 +75,12 @@ resource "azurerm_frontdoor" "example" {
 }
 
 resource "azurerm_frontdoor_custom_https_configuration" "example_custom_https_0" {
-  frontend_endpoint_id              = azurerm_frontdoor.example.frontend_endpoint[0].id
+  frontend_endpoint_id              = azurerm_frontdoor.example.frontend_endpoints["exampleFrontendEndpoint1"]
   custom_https_provisioning_enabled = false
 }
 
```

Comment:
Same here, can we add a blurb at the top of this documentation the calls out the breaking change and why it is required? Maybe something like:

-> **BREAKING CHANGE:** In order to address the ordering issue we have changed the design on how to retrieve existing sub resources such as frontend endpoints, routing rules, etc. Please refer to the updated documentation below for more information.